<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>667</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Password generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourses.qrc">
    <normaloff>:/images/password.png</normaloff>:/images/password.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>376</width>
        <height>651</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Password generator</string>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;rerite code at: &lt;/span&gt;&lt;a href=&quot;https://youtu.be/Si4QgunNhvw?si=KTH5SPd5Y9efsoJFstyle=&quot;&gt;&lt;span style=&quot; font-size:11pt; text-decoration: underline; color:#094fd1;&quot;&gt;Qt. Генератор паролей&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="latinCheck">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Latin letter: abcdefghijklmnopqrstuvwxyz</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="latinCombo">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="currentText">
            <string>All</string>
           </property>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>small letter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>big letter</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cyrillicCheck">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Cyr. let.: абвгдеёжзийклмнопрстуфхцчшщъыьэюя</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cyrillicCombo">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>All letter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>only big</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>only small</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="didgitCheck">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>number (format: start-end)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="didgitEdit">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0-9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="symbolCheck">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>symbol</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="symbolEdit">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>!&quot;#$%&amp;'()*+,-./:;&lt;=&gt;?@[\]^_`{|}~</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Number of characters in the password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="lettersSpin">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>4</number>
           </property>
           <property name="value">
            <number>33</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Number of passwords</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="passCountSpin">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>99</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Create password+entropy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Clear passwords</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Save passwords</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="font">
            <font>
             <family>Al Bayan</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
